src/containers/pages/PlanAssignment/helpers/JurisdictionAssignmentForm/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { ErrorMessage, Field, Form, Formik, FormikActions } from 'formik';
```

Comment:
Looks like this file renders the Organization Select component redundant, if  so; it will hopefully be removed before merge,

Reply:
@p-netm what do you mean?

Reply:
The `OrganizationSelect` component will not be needed anymore, since it served the same purpose

Reply:
Yes you're right.  I chose not to use `OrganizationSelect` because:

1. I do not think that form fields should have access to the redux store.  They just need to work with props imho.
2. I didn't want to refactor it a lot, it would have slowed me down